components/rhs_comment/rhs_comment.jsx

```diff
@@ -132,8 +137,10 @@ class RhsComment extends React.PureComponent {
 
     handleShortcutReactToLastPost = (isLastPost) => {
         if (isLastPost) {
-            const {isReadOnly, channelIsArchived, enableEmojiPicker, post,
-                actions: {emitShortcutReactToLastPostFrom}} = this.props;
+            const {
+                isReadOnly, channelIsArchived, enableEmojiPicker, post,
+                actions: {emitShortcutReactToLastPostFrom},
+            } = this.props;
```

Comment:
That seems like an unrelated change. Does it have to be in this PR?

Reply:
Ohh, that was the autoformatting of my IDE, I will update it